### ISSUE TITLE
Refresh stream if blocked

### ISSUE THREAD

**Issue by SalvatoreAD** [muaz-khan/RTCMultiConnection#325]
Hello I want to know if you can control the stream if it is blocked, do a refresh of the stream during the video-conferencing.

Thanks

**Comment by avspeed** [comment]
blocked as in 'frozen' ? why is it frozen in the first place ? Have you figured that out ? one reason might be the bandwidth that you are using try to lower it see https://github.com/muaz-khan/RTCMultiConnection/wiki/Bandwidth-Management

**Comment by SalvatoreAD** [comment]
Yes when I try to record on firefox, the video of remote stream frozen. Tomorrow I try to low the bandwith

**Comment by SalvatoreAD** [comment]
I could not solve the problem, can anyone help?

**Comment by muaz-khan** [comment]
You can bypass RTCMultiConnection default bandwidth-parameters:

```javascript
connection.processSdp = function (sdp) {
    return sdp; // now Chrome will use default (non-modified) SDP
};
```

**Comment by SalvatoreAD** [comment]
How  can use it?
